setup.cfg

```
@@ -72,14 +72,16 @@ docs =
   astroquery
   jplephem
 
+[options.packages.find]
+exclude = sunpy._dev
 
 [options.entry_points]
 asdf_extensions =
   sunpy = sunpy.io.special.asdf.extension:SunpyExtension
 
 [tool:pytest]
 testpaths = "sunpy" "docs"
-norecursedirs = ".tox" "build" "docs[\/]_build" "docs[\/]generated" "*.egg-info" "examples" "sunpy[/\]cm" ".jupyter" ".history" "tools"
+norecursedirs = ".tox" "build" "docs[\/]_build" "docs[\/]generated" "*.egg-info" "examples" "sunpy[/\]_dev" ".jupyter" ".history" "tools" "sunpy[\/]extern"
```

```suggestion
norecursedirs = ".tox" "build" "docs[\/]_build" "docs[\/]generated" "*.egg-info" "examples" "sunpy[/\]_dev" ".jupyter" ".history" "tools" "sunpy[\/]extern" "sunpy[/\]cm"
```
I think cm is still in 2.0